key/endpoint_test.go

```diff
@@ -49,9 +50,9 @@ func TestNewEndpoint(t *testing.T) {
 		require.Error(err)
 		require.Nil(result)
 
-		knfe, ok := err.(KeyNotFoundError)
+		var keyNotFoundError KeyNotFoundError
```

Comment:
`github.com/stretchr/testify/assert` and `github.com/stretchr/testify/require` both have `ErrorIs` and `ErrorAs` assertions.  you can use those to do these kinds of assertions in just 1-2 lines of code.

Reply:
Thanks for the suggestion! I fixed it.